### PR TITLE
feat: Integrate KardiaChain Network

### PR DESCRIPTION
**Pull request by AJackTi**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
  - [x] I have removed all in development `console.log`s
  - [x] I have removed all commented code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [x] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [x] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
         "jsbi",
         "jsonml",
         "juicebox",
+        "kardiachain",
```
Comment (guanbinrui) [inline cspell.json:124]:
```suggestion
```

File: packages/web3-constants/evm/rpc.json [hunk 2]
```diff
@@ -79,6 +81,8 @@
         "Celo": [0, 0, 0, 0, 0],
         "Fantom": [0, 0, 0, 0, 0],
         "Aurora": [0, 0, 0, 0, 0],
-        "Aurora_Testnet": [0, 0, 0, 0, 0]
+        "Aurora_Testnet": [0, 0, 0, 0, 0],
+        "Conflux": [],
```
Comment (guanbinrui) [inline packages/web3-constants/evm/rpc.json]:
```suggestion
        "Conflux": [0, 0, 0, 0, 0],
```

File: packages/web3-constants/evm/token-list.json [hunk 3]
```diff
@@ -17,6 +17,8 @@
         "Celo": ["https://tokens.r2d2.to/latest/42220/tokens.json"],
         "Fantom": ["https://tokens.r2d2.to/latest/250/tokens.json"],
         "Aurora": ["https://tokens.r2d2.to/latest/1313161554/tokens.json"],
-        "Aurora_Testnet": []
+        "Aurora_Testnet": [],
+        "Conflux": [],
```
Comment (guanbinrui) [inline packages/web3-constants/evm/token-list.json]: Strange! It has value on the `develop` branch. Do you mind merging the `develop` branch?

**Comment by guanbinrui** [comment]
CC @DimensionDev/dev-smartcontract Please deploy our contracts on KaradiaChain.

**Comment by SpaceStation09** [comment]
> CC @DimensionDev/dev-smartcontract Please deploy our contracts on KaradiaChain.

OK. I'll deploy contracts on the new chain later.

File: cspell.json (hunk 1 above)
Comment (AJackTi) [reply cspell.json:124]: Why do you remove it? Could you explain me?

**Comment by peihsendoyle** [comment]
Screen Shot 2022-04-06 at 3 15 00 PM

Sorry but I have a question, why are those files conflicted? We just added Kardiachain network into each file.

File: cspell.json (hunk 1 above)
Comment (guanbinrui) [reply cspell.json:124]: Only one field is necessary.

Comment (AJackTi) [reply cspell.json:124]: Ok bro, thanks for replying to me. I just removed it.

**Comment by guanbinrui** [comment]
Hi @AjackTi, Please add me as a collaborator of you cloned repo. I got some configuration stuff need to update.

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Pull request closed without merging**